src/toil_scripts/gatk_germline/germline_config.py

```diff
@@ -0,0 +1,120 @@
+import textwrap
+
+def generate_config():
+    return textwrap.dedent("""
+        # GATK Germline Pipeline configuration file
+        # This configuration file is formatted in YAML. Simply write the value (at least one space) after the colon.
+        # Edit the values in this configuration file and then rerun the pipeline
+        # Comments (beginning with #) do not need to be removed. Optional parameters may be left blank.
+        ##############################################################################################################
+        # Required: Reference Genome URL (hg19.fa)
+        genome.fa:
+
+        # Optional: Reference Genome Index URL (hg19.fa.fai)
+        genome.fa.fai:
+
+        # Optional: Reference Genome Sequence Dictionary (hg19.dict)
+        genome.dict:
+
+        # Required: URL (1000G_phase1.indels.hg19.sites.fixed.vcf)
+        phase:
```

Comment:
I think we should open a ticket and fix this later, but for all the known sites fields, we should just have `known_indels` and `known_snps` which are arrays.